includes/class-wcpdf-main.php

```diff
@@ -588,6 +590,50 @@ public function init_tmp () {
 		}
 	}
 
+	public function load_custom_and_local_fonts() {
+		require WPO_WCPDF()->plugin_path() . '/vendor/autoload.php';
+
+		$options = array(
+			'defaultFont'				=> 'dejavu sans',
+			'tempDir'					=> $this->get_tmp_path( 'dompdf' ),
+			'logOutputFile'				=> $this->get_tmp_path( 'dompdf' ) . "/log.htm",
+			'fontDir'					=> $this->get_custom_fonts_path(),
+			'fontCache'					=> $this->get_custom_fonts_path(),
+			'isRemoteEnabled'			=> true,
+			'isFontSubsettingEnabled'	=> true,
+			'isHtml5ParserEnabled'		=> true,
+		);
+
+		if ( ! @is_dir( $this->get_dompdf_fonts_path() ) || ! wp_is_writable( $this->get_dompdf_fonts_path() ) ) {
+			$rootDir = $this->get_custom_fonts_path();
+		}
+
+		// save custom fonts
+		$custom_fonts_options = new Options( $options );
+		if( isset( $rootDir ) ) {
+			$custom_fonts_options->setRootDir( $rootDir );
+		}
+		$dompdf = new Dompdf( $custom_fonts_options );
+		$dompdf->getFontMetrics()->saveFontFamilies();
+
+		// save local fonts
+		$options['fontDir']  = $options['fontCache'] = $this->get_tmp_path( 'fonts' );
+		$local_fonts_options = new Options( $options );
+		if( isset( $rootDir ) ) {
+			$local_fonts_options->setRootDir( $rootDir );
+		}
+		$dompdf->getFontMetrics()->setOptions( $local_fonts_options );
+		$dompdf->getFontMetrics()->saveFontFamilies();
+	}
+
+	public function get_custom_fonts_path() {
+		return trailingslashit( WPO_WCPDF()->plugin_path() ) . 'assets' . DIRECTORY_SEPARATOR . 'fonts';
+	}
+
+	public function get_dompdf_fonts_path() {
+		return trailingslashit( WPO_WCPDF()->plugin_path() ) . 'vendor' . DIRECTORY_SEPARATOR . 'dompdf' . DIRECTORY_SEPARATOR . 'dompdf' . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . 'fonts' . DIRECTORY_SEPARATOR;
```

Comment:
we can safely use forward slashes instead of `DIRECTORY_SEPARATOR` and it helps a lot for readability (decreasing the risk of errors).